src/harness/config.js

```diff
@@ -14,7 +13,7 @@ if (USE_LOCAL) {
   fcl.config()
     .put("env", "local")
     .put("accessNode.api", "http://localhost:8080")
-    .put("discovery.wallet", "http://localhost:3000/fcl/authn")
+    .put("discovery.wallet", "http://localhost:8701/fcl/authn")
```

Comment:
@10thfloor did you catch this? Wanted to confirm it's ok to align this with the Docker discovery endpoint so things are consistent